fizzbuss.js

```diff
@@ -1,3 +1,4 @@
+//JavaScript Practice
 for (let number = 1; number <= 99; number++) {
   if (number % 3 === 0 && number % 5 != 0) {
```

Comment:
こちらも他で使用されているのと同様に厳密等価演算子を使用すると良いです！

```suggestion
  if (number % 3 === 0 && number % 5 !== 0) {
```

Reply:
ありがとうございます！修正致しました。